internal/generator/vector/elements/elements.go

```diff
@@ -56,6 +69,23 @@ source = """
 `
 }
 
+func (r RemapCW) Template() string {
+	return `{{define "remapTemplateCW" -}}
+{{if .Desc -}}
+# {{.Desc}}
+{{end -}}
+[transforms.{{.ComponentID}}]
+type = "remap"
+inputs = {{.Inputs}}
+source = """
+.GroupBy = "{{.GroupBy}}"
+.LogGroupPrefix = "{{.LogGroupPrefix}}"
```

Comment:
there is no need to inject these values at run time. these are needed only at conf generation time.